index.html

```diff
@@ -0,0 +1,276 @@
+    <main>
+      <section class="headline" id="homePage">
+        <div class="text">
+          <h3 class="small-title">Welcome to the amazing</h3>
+          <h1 class="big-title">Villa Iluminada Atlixco, Puebla 2021</h1>
+          <p class="paragraph">
+            This tradition started in 2011 and ever since, it has been one of
+            the most awaited events in the locality. People from other states of
+            the country come each December to admire the beauty and the magic of
+            the lights.
+          </p>
+          <p class="last-info">2021.11.27(SAT) ~ 2022.01.06(THU)</p>
+          <p class="last-info">@villailuminada Atlixco,Pue.</p>
+        </div>
+      </section>
+      <section class="program">
+        <h3 class="main">Main Program</h3>
+        <div class="underscore"></div>
+        <section class="row">
+          <div class="atractions">
+            <img src="Images/parade.svg" alt="horse-icon" height="100" />
+            <h4 class="red-title">Parade</h4>
+            <p class="program-paragraph">
+              The event counts with an incredible parade every single night
+              where the magic feels in every corner of this town.
+            </p>
+          </div>
+          <div class="atractions">
+            <img src="Images/christmas-tree.png" alt="tree-icon" height="100" />
+            <h4 class="red-title">Christmas</h4>
+            <p class="program-paragraph">
+              Christmas is the best season for many people, and in this event is
+              a great opportunity to discover the magic of the Christmas.
+            </p>
+          </div>
+          <div class="atractions">
+            <img src="Images/fast-food.svg" alt="food-icon" height="100" />
+            <h4 class="red-title">Food & Drinks</h4>
+            <p class="program-paragraph">
+              The event counts with many different kind of food. We have
+              different stands that offers to you a new opportunity to discover
+              a new taste.
+            </p>
+          </div>
+          <div class="atractions">
+            <img
+              src="Images/souveniers.svg"
+              alt="souveniers-icon"
+              height="100"
+            />
+            <h4 class="red-title">Souveniers</h4>
+            <p class="program-paragraph">
+              In the whole town you are going to discover many different
+              establishments where you can get souveniers.
+            </p>
+          </div>
+          <div class="atractions">
+            <img src="Images/lights.svg" alt="lights-icon" height="100" />
+            <h4 class="red-title">Lights</h4>
+            <p class="program-paragraph">
+              The principal tematic of this event definitely are the lights.
+              Watching the whole town illuminated is the principal reason that
+              many people come in this season every year.
+            </p>
+          </div>
+        </section>
```

Comment:
The items within the _main program_ section of your design do not follow the typography (**font family**, **font-weight**, ...), and do not have appropriate **padding** and **margins** around the items, as well as **spacing** between each item and around the items. Kindly ensure improve upon these:
- [ ] The title _Main Program_  has bigger and bolder text, as can be seen in the screenshot of the original design below.
- [ ] Add appropriate padding on the extreme left and right sides of the contents.
- [ ] Add appropriate **spacing** (margins/ paddings) between and around the items/ contents.
- [ ] The typography(**font sizes**, **font-weight**, **font color**) should closely match the provided template design.
- [ ] Contents of each item are too close in the view

**Your program section:**
![image](https://user-images.githubusercontent.com/12745474/133910059-3b136bc0-45f9-4a95-98b6-a49dbc4ba2b4.png)

![image](https://user-images.githubusercontent.com/12745474/133910141-cb82ead8-2b20-44a8-bf05-79fc18bfa4c4.png)


**The original program section:**
![image](https://user-images.githubusercontent.com/12745474/132545825-150c3b77-7c05-4a8c-aa95-956269677499.png)

![image](https://user-images.githubusercontent.com/12745474/133910160-a93a3dfc-3b8f-41d7-b5e8-73e42a30e0ef.png)

Reply:
- [ ] <b>REQUIRED:</b> The same thing goes here, they are still few things to fix to make this content look a little bit more like the content from the mockup design.

| Original      | Yours |
  | ----------- | ----------- |
| ![scrnli_9_19_2021_4-52-35 AM](https://user-images.githubusercontent.com/45217364/133913847-37f3f2bf-9fac-40c4-9147-f51d1f714a3e.png) | ![scrnli_9_19_2021_4-52-11 AM](https://user-images.githubusercontent.com/45217364/133913854-f3560d19-7e01-4862-8085-2ee2f2a3d9a5.png) |
   - All cards must be of the same size.
   - Everything must be vertically and horizontally centered.
   - Focus on spacing, margins  and elements orientations and make sure that everything reflect the mockup design.